spark/src/main/scala/org/apache/spark/sql/comet/CometWindowExec.scala

```diff
@@ -65,71 +65,27 @@ case class CometWindowExec(
       sparkContext,
       "number of partitions")) ++ readMetrics ++ writeMetrics
 
-  override def supportsColumnar: Boolean = true
-
-  protected override def doExecuteColumnar(): RDD[ColumnarBatch] = {
-    val childRDD = child.executeColumnar()
-
-    childRDD.mapPartitionsInternal { iter =>
-      CometExec.getCometIterator(
-        Seq(iter),
-        getNativePlan(output, windowExpression, partitionSpec, orderSpec, child).get)
-    }
-  }
-
   override def outputOrdering: Seq[SortOrder] = child.outputOrdering
 
   override def outputPartitioning: Partitioning = child.outputPartitioning
 
   protected def withNewChildInternal(newChild: SparkPlan): SparkPlan =
     this.copy(child = newChild)
 
-}
-
-object CometWindowExec {
-  def getNativePlan(
```

Comment:
Because `Window` is native operator, we should chain it in the native operators instead of creating a pseudo scan for it.

Reply:
It also can improve performance as we don't need additional native/JVM bridge.